pt-BR/mail/chrome/messenger-newsblog/newsblog.dtd

```diff
@@ -1,6 +1,6 @@
 <!-- folder pane context menu item -->
-<!ENTITY folderContextRSSSubscribe.label    "Subscribe...">
+<!ENTITY folderContextRSSSubscribe.label    "Assinar...">
```

Comment:
CORRECTIONS NEDEED it should be <!ENTITY folderContextRSSSubscribe.label    "Subscrever...">